Tests/MapboxCoreNavigationTests/CLLocationTests.swift

```diff
@@ -35,7 +35,9 @@ class CLLocationTests: TestCase {
         XCTAssertEqual(location.coordinate.latitude, coordinate.latitude)
         XCTAssertEqual(location.coordinate.longitude, coordinate.longitude)
         XCTAssertEqual(location.coordinate, fixLocation.coordinate)
-        XCTAssertEqual(location.timestamp, fixLocation.time)
+        XCTAssertEqual(location.timestamp.timeIntervalSince1970, fixLocation.time.timeIntervalSince1970,
+                       accuracy: 1e-6)
+        XCTAssertEqual(Int64(location.timestamp.nanosecondsSince1970), fixLocation.monotonicTimestampNanoseconds)
```

Comment:
If the monotonic timestamp is more correct than the normal timestamp, why does this comparison need to be imprecise?

At the moment that Core Location informs the location manager of a location update, the timestamp may not equal the current time exactly. This can be more pronounced whenever Core Location batches updates. Does MapboxNavigationNative account for this difference somehow?

Reply:
@Udumft Would you take a look at this one? Thanks!

Reply:
The change here is not about the precision, but about the meaning of this timestamp. In this version of NN, they've changed how `fixLocation.time` is calculated. This caused our test runs to fail as they were relying on it. In the end, NN advised to rely on monotonic timestamp since it is guaranteed to be reported in chronological order and be strictly related to incoming locations timestamps, while `fixLocation.time` may be, quote: "even run in reverse in some cases".
Accuracy delta in this test, by my understanding, comes from floating points [operations](https://github.com/mapbox/mapbox-navigation-ios/pull/3913/files#diff-d6ffb4f2d925d0d01c410270bc6632a301d5dc925c04057b36ef1e4a9c5ba422R8) done to extract/create int nanoseconds from a floating timestamp.